### DISCUSSION TITLE
Env requiremnts

### DISCUSSION THREAD

**Discussion by nilaan-effectz** [ragapp/ragapp#121] [General]
Can anyone list the env requirements for this 
node version
npm/pnpm version
python version 
etc

**Comment by mrt2081** [comment]
I was able to run with node 18 and python 3.12

**Comment by marcusschiesser** [comment]
Best is to use the same version as in the Dockerfile, see https://github.com/ragapp/ragapp/blob/main/src/ragapp/Dockerfile
